### DISCUSSION TITLE
Scss file changes not updated (parcel-nightly287)

### DISCUSSION THREAD

**Discussion by Cristy94** [parcel-bundler/parcel#4724] [General] [answered]
If I change my `.scss` files, the changes are not applied by parcel. I have to clear `.parcel-cache` before I can test the new changes.

Anyone else having issues with output not updating?

**Comment by mischnic** [comment, marked as answer]
That is a known bug (https://github.com/parcel-bundler/parcel/issues/3708) and rather complex to fix (but we'll get there)

**Comment by Cristy94** [comment]
Thank you for the quick response!

This is a very important problem in my opinion as it's something that's used by many devs and is done pretty often.

**Comment by AyoParadis** [comment]
yeah your right, you have to delete the cache folder. I just ran into the same problem.

**Reply by Cristy94** [reply]
This issue was solved for me in the latest releases.

**Reply by Sky2hawkZ** [reply]
> This issue was solved for me in the latest releases.

Which version exactly? @Cristy94
